docs/apm/machine-learning.asciidoc

```diff
@@ -1,36 +1,62 @@
 [role="xpack"]
 [[machine-learning-integration]]
-===  integration
+=== Machine learning integration
 
 ++++
 <titleabbrev>Integrate with machine learning</titleabbrev>
 ++++
 
-The Machine Learning integration initiates a new job predefined to calculate anomaly scores on APM transaction durations.
-Jobs can be created per transaction type, and are based on the service's average response time.
+The Machine learning integration initiates a new job predefined to calculate anomaly scores on APM transaction durations.
+With this integration, you can quickly pinpoint anomalous transactions and see the health of
+any upstream and downstream services.
 
-After a machine learning job is created, results are shown in two places:
+Machine learning jobs are created per environment, and are based on a service's average response time.
+Because jobs are created at the environment level,
+you can add new services to your existing environments without the need for additional machine learning jobs.
 
-The transaction duration graph will show the expected bounds and add an annotation when the anomaly score is 75 or above.
+After a machine learning job is created, results are shown in two places:
 
+* The transaction duration graph will show the expected bounds and add an annotation when the anomaly score is 75 or above.
++
 [role="screenshot"]
 image::apm/images/apm-ml-integration.png[Example view of anomaly scores on response times in the APM app]
 
-Service maps will display a color-coded anomaly indicator based on the detected anomaly score.
-
+* Service maps will display a color-coded anomaly indicator based on the detected anomaly score.
+If there is no anomalous activity detected,
```

Comment:
Is this fragment with a comma supposed to lead into the screenshot below, or is something missing here? I think something's missing.

Reply:
Yup. Good catch